### ISSUE TITLE
bug: ion-refresher-content, should be able to set aria-label, title or aria-labelledby

### ISSUE THREAD

**Issue by stianmorsund** [ionic-team/ionic-framework#28890]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

v7.x

### Current Behavior

This got my attention due to a failing accessibility test using `@axe-core/playwright`. So the issue that gets warned about is that progressbars should have accessible labels: https://dequeuniversity.com/rules/axe/4.8/aria-progressbar-name?application=playwright

There appears to be no way to set an accessible label when using the `ion-refresher`, like so:

```
<ion-content>
    <ion-refresher slot="fixed" (ionRefresh)="refresh()">
      <ion-refresher-content   ariaLabel="Pull to refresh" />
    </ion-refresher>
  </ion-content>
```

### Expected Behavior

Expect to be able to set a `title`, `aria-label` or `aria-labelledby` as recommended here https://ionicframework.com/docs/api/refresher

### Steps to Reproduce

Just see any of the demos here https://ionicframework.com/docs/api/refresher and observe that the spinners don't have an accessible name

### Ionic Info

Latest version

**Comment by thetaPC** [comment]
@stianmorsund Thank you for the issue!

I have verified that this causes a failing status when it comes to web accessibility.

Are you wanting to customize the text that is passed to `aria-label`,`title` or `aria-labelledby`?

**Comment by stianmorsund** [comment]
@thetaPC Thanks for having a look :) 
Being able to customize the text would be great, then we can also support translations.

**Comment by thetaPC** [comment]
@stianmorsund that makes sense. The team will look into this. Thank you again for submitting the issue!
